libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -610,6 +637,7 @@ bool WaveClip::Paste(double t0, const WaveClip &other)
 
        auto copy = std::make_shared<WaveClip>(other, factory, true);
        copy->ClearSequence(copy->GetSequenceStartTime(), copy->GetPlayStartTime());
+       copy->SetTrimLeft(0);
```

Comment:
Does this added line, with the other at 630, fix a known bug in copy-paste?  Strictly speaking that's not the scope of the PR or the stretching project.  Is there an issue number that it fixes?

Reply:
I will add to the QA checklist anyway

Reply:
The reason is obscure, but there is one.
`copy->ClearSequence(playEndTime, sequenceEndTime)` clears the trimmed data. If we don't call `copy->SetTrimLeft(0)`, one may still think the hidden data is valid.
Further down I call `newClip->ApplyStretchRatio()`. This provisionally sets left and right trim to 0, to reduce the ramping in/out of the effect if possible ...
Note that, in 5-of-6, I remove that call on `newClip`, but this is a healthy change in any case, making changes in the lines below safer.